### ISSUE TITLE
Creating a project fails when any non-standard Gem is added to the project

### ISSUE THREAD

**Issue by LB-JakubSkorupka** [o3de/o3de#4754]
**Describe the bug**
Adding any Gem not included in the Standard template to a new project breaks the creation process. 
The issue occurs when using pre-built SDKs and when building from source.

The following error is displayed in a popup window:

```
Cannot add gem XXX.
Error:
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\builds\\o3de\\build\\windows_vs2019\\bin\\profile\\Code\\enabled_gems.cmake'
At:
  C:\builds\o3de\python\runtime\python-3.7.10-rev2-windows\python\lib\pathlib.py(1072): _raw_open
  C:\builds\o3de\python\runtime\python-3.7.10-rev2-windows\python\lib\pathlib.py(1263): touch
  c:\builds\o3de\scripts\o3de\o3de\enable_gem.py(97): enable_gem_in_project
```

After closing this message another popup window appears: "Failed to configure gems for template.".

Please note that a workaround has been found to create project with custom Gems enabled -
1. Create a new project using the "Standard" template.
2. Build the project.
3. Add the required Gems to the project in the **Project Manager**.
4. Rebuild the project.

**Steps to reproduce**
1. Launch the **Project Manager**
2. Navigate to New Project... -> Create New Project -> Configure with more Gems.
3. Enable any Gem not included in the "Standard" template.
4. Create the configured project.

**Expected behavior**
A project with custom Gems enabled is successfully created.

**Actual behavior**
A project with custom Gems enabled fails to create.

**Video**

https://user-images.githubusercontent.com/86951967/137711546-a81e2050-0c23-41f6-8bff-151197bf58fc.mp4

**Found in Branch**
Development (b58ae15)

**Desktop**
 - Device: PC
 - OS: Windows
 - Version: 10
 - CPU: Intel Core i7-8700K
 - GPU: NVIDIA GeForce RTX 2080 Ti
 - Memory: 16GB

**Comment by AMZN-alexpete** [comment]
This should now be fixed in the stabilization/2110 branch and will likely get merged to development later this week.

**Comment by LB-JakubSkorupka** [comment]
Verified as fixed on Stabilization_2110 (49da85c).
